test/unit/article/controllers/articleController.spec.js

```diff
@@ -84,21 +84,6 @@ describe('Article Controller', () => {
   });
 
   describe('getAllArticles()', () => {
```

Comment:
Also, Emeka, why did you remove the test case that was here initially?

Reply:
Resolved.